charts/clearml-serving/templates/clearml-serving-triton-deployment.yaml

```diff
@@ -0,0 +1,48 @@
+{{ if .Values.clearml_serving_triton.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations: {}
+  labels:
+    clearml.serving.service: clearml-serving-triton
+  name: clearml-serving-triton
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      clearml.serving.service: clearml-serving-triton
+  strategy: {}
+  template:
+    metadata:
+      annotations: {}
+      labels:
+        clearml.serving.network/clearml-serving-backend: "true"
+        clearml.serving.service: clearml-serving-triton
+    spec:
+      containers:
+        - env:
```

Comment:
Can you pls add
```
{{- if .Values.clearml.extraPythonPackages }}
            - name: EXTRA_PYTHON_PACKAGES
              value: '{{ join " " .Values.clearml.extraPythonPackages }}'
            {{- end }}
```
like i did for other deployments?

Reply:
Sure. Is this specific for the triton pod(s) or maybe it makes sense to add the same env variable to all the other deployments?

Reply:
Ok I see that this variable is used in three Dockerfiles: triton, serving, and statistics.  It's probably better to make the extra packages list specific for each deployment?

I'd rather use:

```yaml
{{- if .Values.clearml_serving_triton.extraPythonPackages }}
            - name: EXTRA_PYTHON_PACKAGES
              value: '{{ join " " .Values.clearml_serving_triton.extraPythonPackages }}'
            {{- end }}
```

and change `values.yaml` accordingly:

```yaml
clearml_serving_triton:
  enabled: true
  image: allegroai/clearml-serving-triton
  nodeSelector: {}
  tolerations: []
  affinity: {}
  resources: {}
  extraPythonPackages: []
```

for each of the three deployments. Wdyt?

Reply:
Moving `extraPythonPackages` to specific deployment is a good idea and gives more flexibility that my latest addition, so pls, go for it changing already implemented deployments.